src/components/sendV2/form/form.css

```diff
@@ -7,7 +7,7 @@
   align-items: center;
   box-sizing: border-box;
   width: 100%;
-  max-width: 443px;
+  max-width: 431px;
   height: auto;
   background-color: var(--color-white);
   box-shadow: var(--box-shadow-graph-selector);
```

Comment:
Check on layout if we still use this box-shadow, the box-shadow were removed from multiple places.

Reply:
done